src/pepr/operator/crd/generated/gateway/gateway-v1.ts

```diff
@@ -0,0 +1,981 @@
+/**
+ * Copyright 2025 Defense Unicorns
+ * SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+ */
+
```

Comment:
This file is a duplication of `src/pepr/operator/crd/generated/k8s/gateway-v1.ts` (just a different class name + copyright date). Can we nuke this and use the existing `K8sGateway` instead?

Reply:
Looks like `NetworkGateway` isn't used, so maybe this is just a clean delete.